### PR TITLE
TP9: Documentation and Release Notes Updates

### PR DESCRIPTION
**Pull request by geekspertise**
Here is the Service Mesh TP9 release notes and documentation update. Additional changes may arise after testing is complete. 

Can @brian-avery @tvieira @knrc @JStickler and @dfennessy @vramosp review these docs updates?

### CODE REVIEW

File: servicemesh-install/topics/appendix-threescale-adapter.adoc [hunk 1]
```diff
@@ -22,60 +22,73 @@ Pay particular attention to the `kind: handler` resource. You must update this w
...
+
+== Generating manifests from a deployed adapter
+To generate manifests from a deployed adapter, run the following command replacing the arguments below with the 3scale Admin Portal URL, Service ID, and access token:
+
+```
+oc exec -n istio-system $(oc get po -n istio-system -o jsonpath='{.items[?(@.metadata.labels.app=="3scale-istio-adapter")].metadata.name}') \
+-it -- ./3scale-config-gen \
+--url="https://replace-me.3scale.net:443" --service="example-service-id" --token="access-token"
+```
+This will produce sample output to the terminal and a unique identifier (UID) that must be used for multiple service use cases. Edit these samples if required and create the objects using `oc create` command.
```
Comment (dfennessy) [inline servicemesh-install/topics/appendix-threescale-adapter.adoc]: ...the `oc create` command.

File: servicemesh-release-notes/topics/rn-new-features.adoc [hunk 2]
```diff
@@ -6,8 +6,11 @@ Reason – If known, include why has the enhancement been implemented (use case,
 Result – If changed, describe the current user experience
 ////
 
+= New features Technology Preview 9
+The release of {ProductName} adds support for Kiali 0.15.x, Jaeger 1.11, the 3scale Istio Adapter 0.4.1, and Istio 1.1.0-rc.2.
+
 = New features Technology Preview 8
-The release of {ProductName} adds support for Kiali 0.14.x and the 3scale Istio Adapter 0.3.  
+The release of {ProductName} adds support for Kiali 0.14.x and the 3scale Istio Adapter 0.3.
```
Comment (JStickler) [inline servicemesh-release-notes/topics/rn-new-features.adoc]: Should be Kiali version 0.15.0 and 3Scale 0.4.1.

**Review by JStickler — CHANGES_REQUESTED** [review]
Looks pretty good, just one small change needed.

File: servicemesh-release-notes/topics/rn-new-features.adoc (hunk 2 above)
Comment (geekspertise) [reply servicemesh-release-notes/topics/rn-new-features.adoc]: @JStickler You're looking at the New Features for TP8. The next line up contains the new features for TP9.

Comment (JStickler) [reply servicemesh-release-notes/topics/rn-new-features.adoc]: D'oh!  Never mind then!

File: servicemesh-install/topics/application-requirements.adoc [hunk 3]
```diff
@@ -98,7 +103,7 @@ spec:
 
 [NOTE]
 ====
-{ProductName} does not support manual injection using the upstream `istioctl` command in this release. This release is based on a snapshot of the upstream istio release-1.1 branch. Currently, there is no compatible version of `istioctl` available from the upstream project.
+{ProductName} does not support manual injection using the upstream `istioctl` command in this release. This release is based on a snapshot of the upstream Istio release-1.1 branch. Currently, there is no compatible version of `istioctl` available from the upstream project.
```
Comment (brian-avery) [inline servicemesh-install/topics/application-requirements.adoc]: 1.0.2rc2 has istioctl

Comment (brian-avery) [reply servicemesh-install/topics/application-requirements.adoc]: Also, this is based on a release candidate rather than snapshot.

**Comment by geekspertise** [comment]
@brian-avery @knrc @tvieira @JStickler  I just updated the docs with the changes around Mixer. Can you please take another look at the docs with these changes?

**Comment by brian-avery** [comment]
LGTM

**Comment by tvieira** [comment]
Great, thank you @geekspertise! 
LGTM

**Comment by geekspertise** [comment]
@kalexand-rh and @vikram-redhat TP9 is finally ready to go. Can you merge this PR? It has been peer-reviewed and approved by QE and Dev.

File: servicemesh-install/topics/install.adoc [hunk 4]
```diff
@@ -313,6 +318,29 @@ $ oc new-app -f istio_product_operator_template.yaml --param=OPENSHIFT_ISTIO_MAS
 The OpenShift Master Public URL must be configured to match the public URL of your {product-title} Console, this parameter is required by the Fabric8 Launcher.
 ====
 
+[[update-mixer-policy-enforcement]]
+== Update Mixer policy enforcement
+In previous versions of {ProductName}, Mixer’s policy enforcement was enabled by default. However, starting with {ProductName} {ProductVersion} Mixer policy enforcement is now disabled by default. You must enable it before running policy tasks.
+
+To check the current Mixer policy enforcement status, run the following command:
+```
+oc get cm -n istio-system istio -o jsonpath='{.data.mesh}' | grep disablePolicyChecks
```
Comment (kalexand-rh) [inline servicemesh-install/topics/install.adoc]: All commands need to start with $ or #

File: servicemesh-install/topics/install-supported-config.adoc [hunk 5]
```diff
@@ -10,6 +10,7 @@
 The following are the only supported configurations for the {ProductName} {ProductVersion}:
 
 * Red Hat {product-title} version 3.11.
+* Red Hat {product-title} version 4.0 Beta.
```
Comment (kalexand-rh) [inline servicemesh-install/topics/install-supported-config.adoc]: @vikram-redhat, what do you think about adding 4.0 beta references to the 3.x stream?

Comment (knrc) [reply servicemesh-install/topics/install-supported-config.adoc]: @kalexand-rh My understanding is that the OCP 4 docs are not public so our only option was to include this through the 3.11 docs assuming we continue to publish as part of the OpenShift docs.

Comment (knrc) [reply servicemesh-install/topics/install-supported-config.adoc]: IIRC this was actually the approach suggested by the OpenShift docs team when we raised the issue of there being no public OCP 4 docs

Comment (kalexand-rh) [reply servicemesh-install/topics/install-supported-config.adoc]: @knrc, I checked with Ali, and you're right that it's fine for now. It sounds like the references will be removed at some point after the OCP 4 docs go public.

Comment (knrc) [reply servicemesh-install/topics/install-supported-config.adoc]: Fantastic, thanks very much for confirming

**Comment by JStickler** [comment]
Late to the party but LGTM.

File: servicemesh-install/topics/install-supported-config.adoc (hunk 5 above)
Comment (kalexand-rh) [reply servicemesh-install/topics/install-supported-config.adoc]: Absolutely!

**Comment by kalexand-rh** [comment]
This LGTM. Merging.

**Pull request merged**